R/html_document.R

```diff
@@ -266,6 +266,10 @@ html_document <- function(toc = FALSE,
   else if (!is.null(template))
     args <- c(args, "--template", pandoc_path_arg(template))
 
+  # activate pandoc > 2 features in template
+  if (isTRUE(pandoc2.0()))
```

Comment:
I won't use `isTRUE()` here. If there is anything wrong with the function `pandoc2.0()` (e.g. if it returns `NA`), I'd rather see an error.

Reply:
there is no more `isTRUE`